app/(pages)/staffing/_components/StaffingAdvantages/StaffingAdvantages.tsx

```diff
@@ -38,7 +38,11 @@ export const StaffingAdvantages: FunctionComponent<TProps> = ({
         {description} {!isLaptopS && button}
       </div>
       <div className={imageContainerCN}>
-        <Image src={imageSrc} alt="bubble" />
+        <Image
+          src={imageSrc}
+          alt="bubble"
+          className="md:-mt-[100px] lg:-mt-[100px]"
```

Comment:
```suggestion
          className="md:mt-[100px] lg:mt-[100px]"
```
you can remove this dash before the class, it should work fine, could you confirm please

Reply:
When I remove, styling breaks. We need to use this styling syntax.